MAUI/Cartesian-Charts/Annotation.md

```diff
@@ -0,0 +1,743 @@
+    ...
+
+        <chart:SfCartesianChart.Annotations>
+        
+            <chart:TextAnnotation X1="2" Y1="25" Text="TextT Annotation"/>
+            
+    </chart:SfCartesianChart.Annotations>
+
+    </chart:SfCartesianChart>
+ {% endhighlight %}
+
+{% highlight c# %}
+
+        SfCartesianChart chart = new SfCartesianChart();
+
+        var text = new TextAnnotation()
+        {
+            X1 = 2,
+            Y1 = 25,
+            Text = "Text Annotation"    
+        };
+
+        chart.Annotations.Add(text);
+
+        this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Text annotation in MAUI Chart](Annotation_image/text_annotation.png)
+## Text Customization
+
+The [TextAnnotation]() can be customized using the [LabelStyle]() property. The following properties are used to customize the text:
+
+* [TextColor]() - Changes the text color.
+* [BackgroundColor]() - Changes the background color of the text.
+* [Margin]() - Sets the margin for the text.
+* [Font]() – Changes the text size, font family, and font weight. (This is a deprecated API. Use FontSize, FontFamily, and FontAttributes properties instead.)
+* [FontFamily]() - Changes the font family of the text.
+* [FontAttributes]() - Changes the font style of the text.
+* [FontSize]() - Changes the font size of the text.
+* [HorizontalTextAlignment]() - Aligns the text horizontally at the Start, Center, or End.
+* [VerticalTextAlignment]() - Aligns the text vertically at the Start, Center, or End.
+
+## Shape annotation
+
+The [ShapeAnnotation]() allows you to add annotations in the form of shapes such as rectangles, ellipses, horizontal lines, vertical lines, etc., at specific areas of interest in the chart area.
+
+* [RectangleAnnotation]() - Used to draw a rectangle over the chart area.
+* [EllipseAnnotation]() - Used to draw a circle or an ellipse over the chart area.
+* [LineAnnotation]() - Used to draw a line over the chart area.
+* [VerticalLineAnnotation]() - Used to draw a vertical line across the chart area.
+* [HorizontalLineAnnotation]() - Used to draw a horizontal line across the chart area.
+
+The following APIs are commonly used in all ShapeAnnotation:
+
+* [X2]() - Represents the X2 coordinate of the shape annotation.
+* [Y2]() - Represents the Y2 coordinate of the shape annotation.
+* [Fill]() - Represents the background color of the shape annotation.
+* [Stroke]() - Represents the stroke color of the shape annotation.
+* [StrokeWidth]() - Represents the stroke width of the shape annotation.
+* [StrokeDashArray]() - Represents the stroke dash pattern of the shape annotation.
+* [Text]() - Represents the annotation text of the shape annotation.
+* [LabelStyle]() - Represents the style for customizing the annotation text of the shape annotation.
+
+## Rectangle annotation
+
+The [RectangleAnnotation]() is used to draw a rectangle or a square at specific points in the chart area.
+
+{% tabs %}
+
+ {% highlight xaml %}
+
+        <chart:SfCartesianChart>
```

Comment:
align the code snippet

Reply:
@SaiyathAliFathima , aligned the code snippet